server/replication.go

```diff
@@ -0,0 +1,21 @@
+package server
+
+import (
+	"encoding/binary"
+
+	"github.com/go-mysql-org/go-mysql/mysql"
+)
+
+func parseBinlogDump(data []byte) (*mysql.Position, error) {
+	var p mysql.Position
+	p.Pos = binary.LittleEndian.Uint32(data[0:4])
+	p.Name = string(data[10:])
+
+	return &p, nil
+}
+
+func parseBinlogDumpGTID(data []byte) (*mysql.MysqlGTIDSet, error) {
```

Comment:
do you plan to support MariaDB GTID set? you can see `ParseMariadbGTID`

Reply:
MariaDB doesn't implement COM_BINLOG_DUMP_GTID - so, we can assume that this message sent by MySQL.

https://github.com/MariaDB/server/blob/10.9/include/mysql_com.h#L118

Reply:
Yes, I mean if there's not too much work to do we can support MariaDB by the way. But as you said MariaDB does not use COM_BINLOG_DUMP_GTID so it may not be a trivial work